components/compositing/webview_renderer.rs

```diff
@@ -316,14 +322,26 @@ impl WebViewRenderer {
             return;
         };
 
+        if self.epoch_not_synchronized.get() {
```

Comment:
A side note: The name `dispatch_input_event` seems very misleading. I did a code search and found that it's only used for dispatching simulated mouse events, either when simulating a mouse click, or when handling webdriver-simulated mouse/wheel events.

Could you verify if this is correct?

If so, please rename this function to `dispatch_simulated_input_event`, and rename relevant variables / functions similarly

Please also assert at L321 that we don't receive any event that don't require hit-testing. Otherwise delayed and non-delayed events will be out-of-sync.

Reply:
Thanks! Please add cleanups or add TODO.